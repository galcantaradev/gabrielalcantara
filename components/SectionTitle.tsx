import { ReactNode } from 'react';
import styled from 'styled-components';

interface SectionTitleProps {
  children: ReactNode;
}

const StyledSectionTitle = styled.h2``;

export const SectionTitle = (props: SectionTitleProps) => {
  return <StyledSectionTitle>{props.children}</StyledSectionTitle>;
};
