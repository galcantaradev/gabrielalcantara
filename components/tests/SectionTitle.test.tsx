import { render, screen } from '@testing-library/react';

import { SectionTitle } from '../SectionTitle';

describe('SectionTitle.test.tsx', () => {
  it('should render correctly', () => {
    const { asFragment } = render(<SectionTitle>Title</SectionTitle>);

    expect(screen.getByText('Title')).toBeInTheDocument();
    expect(asFragment()).toMatchSnapshot();
  });
});
