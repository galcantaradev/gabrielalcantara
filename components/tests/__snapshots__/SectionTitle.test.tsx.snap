// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SectionTitle.test.tsx should render correctly 1`] = `
<DocumentFragment>
  <h1
    class="SectionTitle-sc-15x45w6-0 fzzHJf"
  >
    Title
  </h1>
</DocumentFragment>
`;
